templates/express-server/src/api/routes/___.js

```diff
@@ -24,7 +24,7 @@ router.{{@key}}('{{../../subresource}}', async (req, res, next) => {
     {{{quote ../name}}}: req.params['{{../name}}']{{#unless @last}},{{/unless}}
       {{/equal}}
       {{#equal this.in "header"}}
-    {{{quote ../name}}}: req.header['{{../name}}']{{#unless @last}},{{/unless}}
+    {{{quote ../name}}}: req.headers['{{../name}}']{{#unless @last}},{{/unless}}
```

Comment:
You're right, it has to be fixed but I think the right syntax is:

```suggestion
{{{quote ../name}}}: req.get('{{../name}}'){{#unless @last}},{{/unless}}
```

or

```suggestion
{{{quote ../name}}}: req.header('{{../name}}'){{#unless @last}},{{/unless}}
```

See https://expressjs.com/en/api.html#req.get

Reply:
All three seem to give the correct result. But I guess as per the official doc of express, it should be -

```suggestion
    {{{quote ../name}}}: req.headers['{{../name}}']{{#unless @last}},{{/unless}}
    {{{quote ../name}}}: req.get('{{../name}}'){{#unless @last}},{{/unless}}
```

But the syntax req.headers['{{../name}}'] kind of keeps the code looking consistent for all the params.

Reply:
Yeah but it's not documented and I fear they're hiding it as if it were private and thus susceptible to be changed at any time. We should stick to `req.get()` or `req.header()` which are the documented ones. Choose the one you prefer, I have no opinion there 😊

Reply:
Yeah right. Will go for what's documented then.